wagtailapi_legacy/v1/apps.py

```diff
@@ -6,15 +6,15 @@
 
 
 class WagtailAPIAppConfig(AppConfig):
-    name = 'wagtailapi_legacy_v1'
-    label = 'wagtailapi_legacy_v1'
+    name = 'wagtailapi_legacy.v1'
+    label = 'wagtailapi_legacy.v1'
```

Comment:
I think this one should still be an underscore, see https://github.com/wagtail/wagtail/blob/master/wagtail/api/v2/apps.py#L9

Reply:
Makes sense, updated!